elastic/kibana
# chore(slo): Friendlier security exception

Resolves https://github.com/elastic/kibana/issues/181899

## Summary

Small PR to remove the unecessary `security_exception Root causes: security_exception:` reason prefixed


Testing:
1. Create user with role with _not_ enough privileges:
- Create a role with cluster privilege: manage_transform
- `*` index privilege as `read`, `view_metadata`
- `.slo*` index privilege as `all`
- SLO kibana access

2. Log with this user
3. Create an SLO: assert the 403 error is shown correctly.

After | Before
-- | --
![image](https://github.com/elastic/kibana/assets/1376800/bed75d0d-24a4-403f-9a34-bf4c78f3e213) | ![image](https://github.com/elastic/kibana/assets/1376800/55764136-0f15-4329-92ba-b50453c759e7)
